README.rst

```diff
@@ -57,6 +57,27 @@ can find it at `PULL_REQUEST_TEMPLATE.md <.github/PULL_REQUEST_TEMPLATE.md>`_.
 The issue report template should be automatically applied if you are creating an issue on GitHub as well. Otherwise you
 can find it at `ISSUE_TEMPLATE.md <.github/ISSUE_TEMPLATE.md>`_.
 
+Running Schema Builder tool 
+---------------------------
+If you want to run dbt-schema-builder, clone this repository.
+You need a python virtual environment to run this, create and activate a python 3.6 virtual environment using your favorite tool
+
+Inside dbt-schema-builder folder, run the following to update the versions in all of the local requirements files    
+::
+    $ cd dbt-schema-builder
+    $ make upgrade
```

Comment:
Also, I don't think we need to tell people to run `make upgrade` if they're just trying to run schema builder.  That's a maintenance task left for the maintainers (us).

Reply:
Yeah agreed. I think this should be `make requirements`, I always confuse the two.

Reply:
> Sorry, I just noticed we test multiple versions of python (3.6 and 3.8). How come we're recommending to use a specific version (3.6) in the docs?

I saw 3.6 version from this doc [https://openedx.atlassian.net/wiki/spaces/DE/pages/1121059960/dbt+Upgrade+Checklist](url), if I works on multiple versions of python for sure I can change the text form here.
Actually I was just trying to document the steps that I did in order to run dbt-schema-builder (in a similar way that warehouse-transform repository has getting started steps), I wasn't aware of this being a public repository

Reply:
oh yes it should be `make requirements`

Reply:
Thanks for catching all these things in the review, my intent was to document steps for internal team members, but its good to have it here as well from perspective of external users, thanks for detailed review.